<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>465</width>
      <height>311</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabFwUpgrade">
     <property name="font">
      <font>
       <family>Microsoft JhengHei UI</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="MyProgressBar" name="upgProgressBar" native="true">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>211</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddFw">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>65</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>添加固件</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditFwPath">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>281</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUpgEN">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>65</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>更新固件</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelFwVerNewVal">
      <property name="geometry">
       <rect>
        <x>389</x>
        <y>220</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei UI</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0.0.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>150</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelFwVerOld">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>180</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>固件当前版本</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelFwVerNew">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>220</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>固件升级版本</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelFwVerOldVal">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>180</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0.0.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxSerialPortSetting">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>151</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI</family>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>串口设置</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>19</y>
       <width>141</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="spacing">
       <number>1</number>
      </property>
      <item row="0" column="1">
       <widget class="QLabel" name="labelSerialPortNum">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>串口号</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="serialPortBaud">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 8pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QComboBox" name="serialPortStopBit">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 8pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>校验位</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>停止位</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="serialPortNum">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 8pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>数据位</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="serialPortDataBit">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 8pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="serialPortParityBit">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 8pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <item>
         <property name="text">
          <string>NONE</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>波特率</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>流控制</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QComboBox" name="serialPortFlowCtrl">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 8pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <item>
         <property name="text">
          <string>NONE</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="MyBtn" name="btnSerialPort" native="true">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>240</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelSerialPortName">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>50</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>通信设备</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>Fw Verion:1.0.0</string>
    </property>
   </widget>
   <widget class="QWidget" name="logoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <widget class="QLabel" name="labelLogoText">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>2</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(50,254,255)</string>
     </property>
     <property name="text">
      <string>Microphone Charging Case</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelLogoPic">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>2</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMinimize">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>2</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelLogoName">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255,255,255)</string>
     </property>
     <property name="text">
      <string>IAP升级调试工具</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClose">
     <property name="geometry">
      <rect>
       <x>605</x>
       <y>2</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>360</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(50,254,255)</string>
    </property>
    <property name="text">
     <string>深圳弧克创新科技有限公司</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>764</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyBtn</class>
   <extends>QWidget</extends>
   <header location="global">mybtn.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyProgressBar</class>
   <extends>QWidget</extends>
   <header>myprogressbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
